corehq/motech/repeaters/tasks.py

```diff
@@ -41,6 +42,20 @@
 logging = get_task_logger(__name__)
 
 
+@periodic_task(
+    run_every=crontab(day_of_month=27),
```

Comment:
I'm pretty sure this means every minute for the whole day of the 27th day of the month

Reply:
https://github.com/dimagi/commcare-hq/pull/29259